downstream/modules/automation-hub/proc-downloading-signature-public-keys.adoc

```diff
@@ -0,0 +1,27 @@
+//this module appears in assembly-collections-and-content-signing-in-pah
+
+[id="proc-downloading-signature-public-keys"]
+
+= Downloading signature public keys
+
+After you sign and approve collections, you must download the signature public keys from the {HubName} UI before you add it to the local system keyring.
+
+.Procedure
+
+. Log in to your private {HubName} instance in the {HubName} UI.
+. In the left navigation, click *Signature Keys*. The Signature Keys dashboard displays a list of multiple keys: collections and container images.
+
+. Select the menu icon, click btn:[Download Key] to download the public key.
++
```

Comment:
Should this verification step be its own step and not a sub-step to downloading the public key?